internal/querynodev2/delegator/idf_oracle.go

```diff
@@ -483,14 +513,14 @@ func (o *idfOracle) SyncDistribution() error {
 		if ok && activate && !statsActivate {
 			stats, err := stats.FetchStats()
 			if err != nil {
-				rangeErr = err
+				rangeErr = fmt.Errorf("fetch stats failed with error: %v", err)
```

Comment:
I did not look closely at this piece of logic, please check https://github.com/zhengbuqian/milvus/pull/27#discussion_r2362435239 and see if it makes sense

Reply:
> I did not look closely at this piece of logic, please check [zhengbuqian#27 (comment)](https://github.com/zhengbuqian/milvus/pull/27#discussion_r2362435239) and see if it makes sense

I don't make sure if we could exist unreadable sealed segment before first target.
Now we will keep unreadable segment in current till we remove it from snapshot.
If we remove snapshot before first target, here will be correct.
But we should add protection.